client/coral-framework/graphql/mutations/index.js

```diff
@@ -10,16 +10,59 @@ export const postComment = graphql(POST_COMMENT, {
   options: () => ({
     fragments: commentView
   }),
-  props: ({mutate}) => ({
-    postItem: ({asset_id, body, parent_id} /* , type */ ) => {
-      return mutate({
+  props: ({ownProps, mutate}) => ({
+    postItem: ({asset_id, body, parent_id}) =>
+      mutate({
         variables: {
           asset_id,
           body,
           parent_id
+        },
+        optimisticResponse: {
+          createComment: {
+            comment: {
+              user: {
+                id: ownProps.auth.user.id,
+                name: ownProps.auth.user.username
+              },
+              created_at: new Date(),
+              body,
+              parent_id,
+              asset_id,
+              action_summaries: [],
+              tags: [],
+              status: null,
+              id: `${Date.now()}_temp_id`
+            }
+          }
+        },
+        updateQueries: {
+          AssetQuery: (oldData, {mutationResult:{data:{createComment:{comment}}}}) =>
+
+            // If posting a reply
+            parent_id ? {
```

Comment:
I love es6 as much as the next guy, but can we get explicit return statements, especially for returning a ternary of objects? I don't want to abuse the arrow syntax.